packages/@aws-cdk/aws-applicationautoscaling/lib/scalable-target.ts

```diff
@@ -225,6 +226,14 @@ export interface ScalingSchedule {
    * @default No new maximum capacity
    */
   readonly maxCapacity?: number;
+
+  /**
+   * The time zone used when referring to the date and time of a scheduled action,
+   * when the scheduled action uses an at or cron expression.
+   *
+   * @default - No specific time zone.
```

Comment:
I think we can do better with this default. Presumably if you specify a scheduled action it will be referring to the date/time of _some_ timezone. that should be the default here.